simpleclient_jetty/src/main/java/io/prometheus/client/jetty/JettyStatisticsCollector.java

```diff
@@ -1,18 +1,17 @@
 package io.prometheus.client.jetty;
```

Comment:
I don't want to depend one module to another. In this way, we can get rid of the older one in the future.
Also, in the case I remove this class in jetty module, if someone wants to add jetty_jdk8 dependendency, QueuedThreadPoolStatisticsCollector will be seen as duplicate. It would be also confusing. 
What is your suggestion about this?